### ISSUE TITLE
RC Research: Ratio and Proportion 1.1.0-rc.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#647]
@BLFiedler @emily-phet Ratio and Proportion 1.1.0-rc.1 is ready for RC Research testing. The link is needed by no later that next Thursday, 5/6. This is an internal data-recording study to be conducted by @BLFiedler. By my understanding, recording will just be done remotely with just a couple of student. Document issues in https://github.com/phetsims/ratio-and-proportion/issues and link to this
issue. 

In general the hope is that there would be no more than 1 hour of testing done for this. The sim was just published, and seems pretty stable. We just mainly want to make sure that the recording endpoint and Metacog full stack is working with this sim. Please just make sure that the basics of the sim works. 

Unsure, but testing on chrome/windows and mac/safari feels like enough to me.

It is NOT expected that the playback wrapper will work perfectly with keyboard navigation. This is alright, as we will be looking at data stream directly, and not the playback wrapper. Mostly this is trying to note that we already know about some problems, like https://github.com/phetsims/phet-io/issues/1748

Assigning to @kathy-phet and @KatieWoe for prioritization.

<details>
<summary><b>PhET-iO RC-research Test</b></summary>

<h3>What to Test</h3>

- Make sure standalone sim is working properly.
- Try to break the PhET-iO sim.
- Try to break the PhET-iO sim using alternative input, as this sim has Interactive Description support.
- Please poke around in Studio briefly on just one device.
- links should be made public, but I will do that after testing. 
- Conduct a recording test to Metacog, further instructions in the QA Book. 

<h3>Focus and Special Instructions</h3>

- I want to emphasize that only minimal testing is required for this. 

<h3>Link(s)</h3>

- **[Wrapper Index](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.1.0-rc.1/phet-io)**

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
QA is done with the quick check. Some keyboard nav inputs seem to be ignored by the recording, so I may not have found all instances.

**Comment by zepumph** [comment]
Thanks! This is so helpful. I really appreciate it.
